### PR TITLE
CVE-2022-30190 - MSDT RCE

### PR DESCRIPTION
**Pull request by melamedbn**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
New pack for CVE-2022-30190

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [x] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Review by tomer-pan — APPROVED** [review]

File: Packs/CVE_2022_30190/Playbooks/playbook-CVE-2022-30190_-_MSDT_RCE_README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+On **May 27th**, a new Microsoft Office Zero-Day was discovered by Nao_sec. 
+
+The new Zero-Day is a remote code execution vulnerability that exists when MSDT is called using the URL protocol from a calling application such as Word. 
+
+On **May 30th**, Microsoft assigned **CVE-2022-30190** to the MSDT vulnerability, aka **Follina vulnerability**.
+
+The playbook includes the following tasks:
```
Comment (michalgold) [inline Packs/CVE_2022_30190/Playbooks/playbook-CVE-2022-30190_-_MSDT_RCE_README.md:7]: This playbook includes the following tasks:
* Collect detection rules.
* Exploitation patterns hunting using Cortex XDR - XQL Engine and 3rd party SIEM products.
* Cortex XDR BIOCs coverage.
* Provides Microsoft workarounds and detection capabilities.

File: Packs/CVE_2022_30190/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+On **May 27th**, a new Microsoft Office Zero-Day was discovered by Nao_sec.
```
Comment (michalgold) [inline Packs/CVE_2022_30190/README.md:1]: Let's add first
This pack is part of the [Rapid Breach Response (https://xsoar.pan.dev/marketplace/details/MajorBreachesInvestigationandResponse) pack.

File: Packs/CVE_2022_30190/Playbooks/playbook-CVE-2022-30190_-_MSDT_RCE_README.md (hunk 1 above)
Comment (melamedbn) [reply Packs/CVE_2022_30190/Playbooks/playbook-CVE-2022-30190_-_MSDT_RCE_README.md:7]: Done

File: Packs/CVE_2022_30190/README.md (hunk 2 above)
Comment (melamedbn) [reply Packs/CVE_2022_30190/README.md:1]: Done

**Pull request merged**